librz/include/rz_endian.h

```diff
@@ -591,37 +592,68 @@ static inline void rz_write_ble_double(void *src, double val, bool big_endian) {
 	big_endian ? rz_write_be_double(src, val) : rz_write_le_double(src, val);
 }
 
+#ifndef __has_builtin
```

Comment:
You forgot to remove this

Reply:
I'm afk for a few hours, I'll get it fixed asap.

Reply:
Done